File: 4-Patron/php/SkinOnline.php
<?php
class SkinOnline
{
  private $infoSkin;
  private $texturaSkin;

  function __construct($info, $txs)
  {
    $this->infoSkin = $info;
    $this->texturaSkin = $txs;
  }

  function getInfoSkin()
  {
    return $this->infoSkin;
  }

  function setInfoSkin($nInfo)
  {
    $this->infoSkin = $nInfo;
  }

  function getTextura()
  {
    return $this->texturaSkin;
  }

  function setTextura($nTextura)
  {
    $this->texturaSkin = $nTextura;
  }

  function mostrarInfoSkin()
  {
    $infSkin = $this->infoSkin->getInfoSkin();
    return  "Autor: " . $infSkin[0] .
            "Fecha: " . $infSkin[1].
            "Descripción: " $infSkin[2] .
  }

  function mostrarTexturaSkin()
  {
    $this->texturaSkin->mostrarTextura();
  }
}
?>

File: 4-Patron/python/SkinOnline.py
class SkinOnline:
    def __init__(self, info, textura):
        self.__infoSkin = info
        self.__texturaSkin = textura

    def getInfoSkin(self):
        return self.__infoSkin

    def setInfoSkin(self, nInfo):
        self.__infoSkin = nInfo

    def getTextura(self):
        return self.__texturaSkin

    def setTextura(self, nTextura):
        self.__texturaSkin = nTextura

    def mostrarInfoSkin(self):
        infSkin = self.__infoSkin.getInfoSkin()
        return f"Autor: . {infSkin[0]} . \
                Fecha: . {infSkin[1]}. \
                Descripción: {infSkin[2]}. "

    def mostrarTexturaSkin(self):
        self.__texturaSkin.mostrarTextura()
